src/pytest_dynamodb/factories.py

```diff
@@ -141,10 +141,6 @@ def dynamodb_factory(request):
                 host=proc_fixture.host,
                 port=proc_fixture.port
             ),
-            # these args do not matter (we have to put something to them)
-            region_name='us-east-1',
```

Comment:
have these as configurable options for regular trio. fixture factory, pytest command line args and pytest ini. 

We should be able to configure both presence or the lack of these options as well.